src/lib/core/utils/data-element-constraints.ts

```diff
@@ -139,12 +143,16 @@ export function flattenConstraints(
       const passesMaxValuesConstraint =
         constraint.maxNumValues === undefined ||
         constraint.maxNumValues >= variable.distinctValuesCount;
+      const passesTemporalConstraint =
+        isEmpty(constraint.isTemporal) ||
+        constraint.isTemporal == variable.isTemporal;
```

Comment:
This means that, if `isTemporal` is set for the variable constraint, then it **must** also be set on the variable for it to pass. If the constraint contains `isTemporal = false`, would a variable without `isTemporal` set be considered a viable candidate?

Reply:
hmm. i had assumed, perhaps erroneously, that isTemporal would always be available for a variable. i figured if it was annotated as being temporal that value would be true, false otherwise. is that not the case?